libs/ha/ha_common_libs_k8s.py

```diff
@@ -1326,11 +1350,21 @@ def simulate_disk_cvg_failure(self, node_obj, source, resource_status, resource_
         ha_pod = node_obj.get_all_pods(pod_prefix=common_const.HA_POD_NAME_PREFIX)[0]
         LOGGER.info("Publishing event %s for %s resource type through ha pod %s", resource_status,
                     resource_type, ha_pod)
-        cmd = common_cmd.PUBLISH_CMD.format(common_const.MOCK_MONITOR_REMOTE_PATH, config_json_file)
+        resp = node_obj.copy_file_to_remote(local_path=config_json_file,
+                                            remote_path=common_const.HA_CONFIG_FILE)
```

Comment:
We have to copy config.json inside HA pod, please add :
LOGGER.info("Copying file inside pod %s", ha_pod)
            node_obj.execute_cmd(common_cmd.HA_COPY_CMD.format(
                common_const.HA_CONFIG_FILE, ha_pod,
                common_const.HA_CONFIG_FILE))